lib/datasets/Flights.py

```diff
@@ -1,2 +1,58 @@
-def dummy_dataset():
-    return [1, 2, 3, 4]
+import pandas as pd
+
+
+def flights_dataset():
+    df = pd.read_csv('flights_data_002.csv')
+    df = df.drop(['conversion'], axis=1)
```

Comment:
This is a massive PR. It would be better to do a PR for every feature that is added to the repo